tests/topology/test_pbc_cell_div.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Sep 12 18:49:53 2019
+
+@author: georgecourcoubetis
+"""
+
+import time
+
+start = time.time()
+
+from tyssue import stores
+from pathlib import Path
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import sys, os
+
+#%matplotlib inline
+# from tyssue.generation import hexa_grid2d
+from tyssue import config, Sheet
+from tyssue import SheetGeometry, PlanarGeometry
+from tyssue.io import hdf5
+
+from tyssue.draw.plt_draw import quick_edge_draw
+from tyssue.draw import sheet_view
+
+from tyssue.topology.sheet_topology import remove_face, cell_division
+from tyssue.solvers.quasistatic import QSSolver
+from tyssue.dynamics.planar_vertex_model import PlanarModel as model
```

Comment:
I don't think we need to use a model here, just performing division w/o any gradient descent should suffice to test if the code is working

Reply:
As you wish!